packages/client/hmi-client/src/components/operator/tera-operator-menu.vue

```diff
@@ -0,0 +1,77 @@
+<template>
+	<section ref="operatorMenu" :style="style">
+		<Button
+			v-if="node"
```

Comment:
Why the `v-if="node"`? The component should not be visible if there is no node right?